docking-station-app/src/components/ActionView/ActionMenu/index.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+
+import { useListComposeStacks } from '@/hooks/stacks'
+import { useFiltersStore } from '@/store'
+import {
+  Burger,
+  Center,
+  Group,
+  Menu,
+  useMantineColorScheme,
+} from '@mantine/core'
+import { IconCloudDownload, IconRefresh } from '@tabler/icons-react'
+import { useState } from 'react'
+import ThemeButton from '../../ThemeButton'
+
+export default function ActionMenu({ onUpdate, onRefresh }: { onUpdate: () => void, onRefresh: () => void }) {
+  const { colorScheme } = useMantineColorScheme();
+  const [menuOpened, setMenuOpened] = useState(false);
+  const { data: stacks = [] } = useListComposeStacks({
+    enabled: false,
+    meta: { noCache: true },
+  })
+  const services = stacks.flatMap(({ services }) => services)
+  const [selectedServicesKeys] = useFiltersStore((state) => [state.selectedServices])
+  const selectedServices = (services.filter(({ stackName, serviceName }) => (
+    !selectedServicesKeys.size
+    || selectedServicesKeys.has(`${stackName}/${serviceName}`)
+  ))
+  )
+  const selectedServicesWithUpdates = (selectedServices.filter((service) => service.hasUpdates))
+  return (
+    <Center component={Group} gap={5} wrap='nowrap'>
+      <Menu withArrow transitionProps={{ transition: 'scale-y', duration: 100 }} onOpen={() => setMenuOpened(true)} onClose={() => setMenuOpened(false)}>
+        <Menu.Target>
+          <Burger c={colorScheme == 'dark' ? 'gray.7' : 'gray.4'} opened={menuOpened} aria-label='Menu' />
```

Comment:
```suggestion
          <Burger color={colorScheme === 'dark' ? 'gray.4' : 'gray.7'} opened={menuOpened} aria-label='Menu' />
```

Don't ask me why its `color={...}` and not `c={...}`, I have no idea.